docs/schema.md

```diff
@@ -270,7 +299,32 @@ Representation of an AWS [IAM Role](https://docs.aws.amazon.com/IAM/latest/APIRe
     (AWSAccount)-[AWS_ROLE]->(AWSRole)
     ```
 
+## AWSVpc
+Representation of an AWS VPC configuration
```

Comment:
Adding the link to the datatype: 
`Representation of an AWS [VPC configuration](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Vpc.html)`